addOns/bruteforce/src/main/resources/org/zaproxy/zap/extension/bruteforce/resources/Messages.properties

```diff
@@ -13,6 +13,7 @@ bruteforce.dir.and.children.popup   = Forced Browse Directory (and Children)
 bruteforce.options.button.addfile	= Select File...
 bruteforce.options.error.failCaseString.invalid = The Fail Case String should not be empty.
 bruteforce.options.label.addfile	= Add custom Forced Browse file:
+bruteforce.options.label.browsefileswithoutextension = Force Browse files without extension and without slash at the end
```

Comment:
I'd remove the `and without slash...`, being a file already implies not having a slash.